### PR TITLE
Adds roll-able sleeves, switchable eyepatch.

### PR DESCRIPTION
**Pull request by Datraen**
See #834, icons courtesy of RedStryker.

### CODE REVIEW

File: code/modules/clothing/clothing.dm [hunk 1]
```diff
@@ -718,14 +742,45 @@ BLIND     // can't see anything
...
 		usr << "<span class='notice'>You cannot roll down [src]!</span>"
 		return
+	if((rolled_sleeves == 1) && !(rolled_down))
+		rolled_sleeves = 0
 
 	rolled_down = !rolled_down
 	if(rolled_down)
-		body_parts_covered &= LOWER_TORSO|LEGS|FEET
+		body_parts_covered = initial(body_parts_covered)
+		body_parts_covered &= !(UPPER_TORSO|ARMS)
```
Comment (PsiOmegaDelta) [inline code/modules/clothing/clothing.dm]: ~(UPPER_TORSO|ARMS)

**Comment by Datraen** [comment]
Thanks Psi.

**Comment by comma** [comment]
Kinda belatedly (after you already gone around added them) but would've been easier if you just set base undersuit's class rolled_sleeves to -1

**Comment by Datraen** [comment]
Base undersuit is set to one.
Glasses are commented and completely defined in the glasses dm

**Pull request merged**